include/unifex/retry_when.hpp

```diff
@@ -346,36 +346,26 @@ class _sender<Source, Func>::type {
         std::is_constructible_v<std::remove_cvref_t<Receiver>, Receiver> &&
         is_connectable_v<Source&, source_receiver<Source, Func, std::remove_cvref_t<Receiver>>>, int> = 0>
   operation<Source, Func, Receiver> connect(Receiver&& r) &&
-      noexcept(std::is_nothrow_constructible_v<
-          operation<Source, Func, Receiver>, Source, Func, Receiver>) {
+      noexcept(
+        std::is_nothrow_move_constructible_v<Source> &&
+        std::is_nothrow_move_constructible_v<Func> &&
+        std::is_nothrow_constructible_v<std::remove_cvref_t<Receiver>, Receiver>) {
```

Comment:
Needs to also check `is_nothrow_connectable_v<Source&, source_receiver<Source, Func, std::remove_cvref_t<Receiver>>>` in the `noexcept` clause.

Reply:
And also in the `connect() const &`  noexcept clause below.